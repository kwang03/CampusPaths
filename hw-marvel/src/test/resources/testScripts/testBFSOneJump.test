#Tests the BFS when the shortest path is one jump away

LoadGraph myGraph staffSuperheroes.csv

FindPath myGraph Ernst-the-Bicycling-Wizard Perkins-the-Magical-Singing-Instructor