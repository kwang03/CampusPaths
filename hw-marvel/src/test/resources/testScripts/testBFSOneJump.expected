#Tests the BFS when the shortest path is one jump away

loaded graph myGraph

path from Ernst-the-Bicycling-Wizard to Perkins-the-Magical-Singing-Instructor:
Ernst-the-Bicycling-Wizard to Perkins-the-Magical-Singing-Instructor via CSE331